starter-code/app.js

```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const server = express();
+const hbs = require("hbs")
+const path = require("path");
+const port = 4295;
+
+
+
+// INITIAL SETUP
+
+server.use(express.static(path.join(__dirname, "public")));
+
+server.set("views", path.join(__dirname, "views"));
+
+server.set("view engine", "hbs");
+
+hbs.registerPartials(path.join(__dirname, "views/partials"));
+
+// END OF SETUP
+/* ***** */
+
+// ROUTES
+
+server.get("/", (req, res) => {
+    const data = {
+        definition: " i.e. Street style is a specific fashion style which comes originally from British fashion culture. It is a comprehensive approach to fashion and produces styles which intersect and differ from mainstream fashion considerations.",
+
+        myThoughts: "fashion have always been a way to express oneself but over time, some rules was established by grand figures from this world. street means freedom. a place where no one is enough qualified to give a judgement on anything. which is perfect for those with originality, creativity. street style is a bottomless pit full of unlimited and limitless outfit. inspiration from everyone and for everyone.",
+    }
+
+    res.render("home.hbs", data);
```

Comment:
You don't need to be specific about the type of file, you can ommit the ".hbs" at the end of the file since you have defined 
hbs as the default view engine, so it will assume it is an hbs file.